Clients/src/App.tsx

```diff
@@ -22,7 +22,7 @@ function App() {
       <CssBaseline />
       <Routes>
         <Route path="/" element={<Dashboard />}>
-          <Route path="/" element={<Home />} />
+          <Route path="/home" element={<Home />} />
```

Comment:
No need to re-change `"/home"` to `"/"` to what it was.
This `<Home />` page is the default one that the user must see as the default "/" route